### PR TITLE
Va.gov 526 reroute cv

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
This PR does a few things:
1) Changes all `/disability-benefits/apply/form-526-disability-claim` links to `/disability/how-to-file-claim/`
2) Removes the eBenefits button from the VA.gov page `/disability/how-to-file-claim/`
3) Fixes spacing on the wizard
4) Changes the flag gating the wizard from `__BUILDTYPE__` to the brand consolidation feature flag

## Testing done
Fired up `yarn watch --brand-consolidation-enabled` to see the WBC build and checked the home page to make sure the right link was changed, then checked to make sure the eBenefits button was gone and the spacing looked good.

Fired up `yarn watch` (no brand consolidation flag) and made sure the wizard didn't show up but the eBenefits button did.

## Screenshots
Before spacing fix:
image

After spacing fix:
image

## Acceptance criteria
- [x] The links are correct
- [x] The eBenefits button is removed
- [x] The wizard is only shown when the brand consolidation flag is present

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: va-gov/layouts/home.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h3 class="hub-links-title"><i class="icon-large fa fa-handshake-o hub-color-dis
             <li><a href="/claim-or-appeal-status">Check your claim or appeal status</a></li>
             <li><a href="/va-payment-history/">View your payment history</a></li>
             <li><a href="/disability/upload-supporting-evidence/">Upload evidence to support your claim</a></li>
-            <li><a href="/disability-benefits/apply/form-526-disability-claim">File for a VA disability increase</a></li>
+            <li><a href="/disability/how-to-file-claim/">File for a VA disability increase</a></li>
             <li><a href="/disability/how-to-file-claim/">File a claim for compensation</a></li>
```
Comment (ncksllvn) [inline va-gov/layouts/home.html]: Should these two links both be `/disability/how-to-file-claim/`?

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: va-gov/layouts/home.html (hunk 1 above)
Comment (cvalarida) [reply va-gov/layouts/home.html]: I asked the same question, and @djmunoz1 said yes. :man_shrugging:

**Comment by cvalarida** [comment]
@ncksllvn I made a couple changes after you reviewed. Can you double-check my use of the brand consolidation flag? I _works_ so I think I did it right, but a pair of eyes that knows how the thing should work would be great!

**Review by markgreenburg — APPROVED** [review]

File: src/applications/static-pages/static-pages-entry.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ if (burialPages.has(location.pathname)) {
   });
 }
 
-if (disabilityPages.has(location.pathname) && __BUILDTYPE__ !== 'production') {
+if (disabilityPages.has(location.pathname) && brandConsolidation.isEnabled()) {
   createDisabilityIncreaseApplicationStatus(store);
 }
```
Comment (ncksllvn) [inline src/applications/static-pages/static-pages-entry.js]: This should be enabled on all VA.gov domains, but not on Vets.gov? We can possibly just remove the Vets.gov page paths from the list of Disability pages. Either way, looks good 👍

**Review by ncksllvn — APPROVED** [review]

File: src/applications/static-pages/static-pages-entry.js (hunk 2 above)
Comment (cvalarida) [reply src/applications/static-pages/static-pages-entry.js]: Right, exactly. I made sure with @djmunoz1 before making the change. Vets.gov will have the `Apply on eBenefits` button, but VA.gov will have the find-your-form wizard (which is what this component is).

**Pull request merged**